wca-regulations.md

```diff
@@ -148,7 +148,7 @@ Note: Because Article and Regulation numbers are not reassigned when Regulations
     - 3k2) Penalty for attempts done with puzzles which are not permitted: disqualification of the attempt (DNF). The following exceptions apply:
         - 3k2a) If a non-permitted puzzle is found before a round is complete, affected results in the round may be replaced with extra attempts, at the discretion of the WCA Delegate.
         - 3k2b) 3x3x3 Multi-Blind: if any puzzles are found to be non-permitted, such puzzles may be individually counted as unsolved (without disqualifying the entire attempt), at the discretion of the WCA Delegate.
-- 3l) A puzzle may have a logo on a colored part. If it does, it must have at most one colored part with a logo. Exception: For blindfolded events, a puzzle must not have a logo.
+- 3l) A puzzle may have a logo or multiple logos on a colored part. If it does, it must have at most one colored part with a logo. Exception: For blindfolded events, a puzzle must not have a logo.
```

Comment:
```suggestion
- 3l) A puzzle may have one or more logos on a colored part. If it does, it must have at most one colored part with a logo. Exception: For blindfolded events, a puzzle must not have a logo.
```

Reply:
```suggestion
- 3l) A puzzle may have one or more logos on a colored part. If it does, it must have at most one colored part with logos. Exception: For blindfolded events, a puzzle must not have a logo.
```
Alternatively "a logo or logos", although it looks like the plural can refer to the singular elsewhere in the regulations (e.g. E3c, 5a), so this should be sufficient).

Reply:
I'm still in favour of "any logos" for the last line (BLD). "must not have logos" sounds more odd to me.